unit_test/tstGridOperator.hpp

```diff
@@ -249,8 +249,8 @@ void gatherScatterTest()
     auto bar_out_host = Kokkos::create_mirror_view_and_copy(
         Kokkos::HostSpace(), fm->view( FieldLocation::Cell(), BarOut() ) );
     Cajita::grid_parallel_for(
-        "check_grid_out", Kokkos::Serial(), *( mesh->localGrid() ),
-        Cajita::Own(), Cajita::Cell(),
+        "check_grid_out", Kokkos::DefaultHostExecutionSpace(),
```

Comment:
Is this for loop check thread safe? I guess if `Serial` wasn't enabled then this could be `OpenMP`

Reply:
It should be - each index is only accessed by one thread once. I added this since we test each host backend without `Serial` enabled